src/txn/txn.c

```diff
@@ -1884,6 +1909,16 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
     txn->prepare_count = 0;
 #endif
 
+    /* Add a 2 second wait to simulate commit transaction slowness. */
+    tsp.tv_sec = 2;
+    tsp.tv_nsec = 0;
+    __wt_timing_stress(session, WT_TIMING_STRESS_COMMIT_TRANSACTION_SLOW, &tsp);
+
+    if (!prepare) {
```

Comment:
We need a comment here explain how this should work.

Reply:
Added the comment in 9ef616b393614af333f6c1f0660018ebab2effef